### DISCUSSION TITLE
rendered views for todo and asciidoc?

### DISCUSSION THREAD

**Discussion by Yellow-Subroutine** [gsantner/markor#2008] [General] [answered]
Hi,

just discovered markor and it's great für markdown, where you have an "edit" view and a "rendered" view.

However when you use the todo.txt format or asciidoc all you get in the rendered view is plain text without any highlighting and now I wonder (as I cannot see the point in that):

Is that the way it is supposed to work, is that part of the app not quite finished yet or am I doing something wrong?

What would I have to do to eg. get a table in asciidoc rendered as a table?

**Comment by harshad1** [comment]
No one has found the time to add highlighting for the 'rendered' todo view. It also does not make much sense for todo. Instead almost all todo actions happen in the edit mode. Click on the document title for advanced filtering.

**Comment by gsantner** [comment, marked as answer]
Please watch  #2022 #1880
